Sources/HTMLKit/Internal/Core/Builders/ContentBuilder.swift

```diff
@@ -27,10 +27,6 @@
         return components
     }
     
-    public static func buildBlock(_ components: [T], _ trailing: T...) -> [T] {
```

Comment:
After adding buildArray function and using for..in in the result build block, the compiler will complain "Ambiguous use of 'buildBlock'" and will found 2 candidates (Line 26 - 28 and Line 30 - 32)